deploy/helm/tests/__snapshot__/logs-daemon-set-windows_test.yaml.snap

```diff
@@ -0,0 +1,106 @@
+DaemonSet spec for windows nodes should match snapshot when using default values:
+  1: |
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/os
+                  operator: In
+                  values:
+                    - windows
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+                - key: eks.amazonaws.com/compute-type
+                  operator: NotIn
+                  values:
+                    - fargate
+    containers:
+      - command:
+          - c:\swi-otelcol.exe
+          - --config=c:\conf\relay.yaml
+        env:
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: SOLARWINDS_API_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: SOLARWINDS_API_TOKEN
+                name: solarwinds-api-token
+                optional: true
+        envFrom:
+          - configMapRef:
+              name: RELEASE-NAME-swo-k8s-collector-common-env
+        image: solarwinds/swi-opentelemetry-collector:0.8.3-nanoserver-ltsc2022
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 13133
+        name: swi-opentelemetry-collector
+        ports:
+          - containerPort: 8888
+            name: http
+            protocol: TCP
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 13133
+        resources:
+          limits:
+            memory: 700Mi
+          requests:
+            memory: 700Mi
+        volumeMounts:
+          - mountPath: c:\var\log\pods
+            name: varlogpods
+            readOnly: true
+          - mountPath: c:\var\log\containers
+            name: varlogcontainers
+            readOnly: true
+          - mountPath: c:\var\lib\docker\containers
+            name: varlibdockercontainers
+            readOnly: true
+          - mountPath: c:\conf
+            name: opentelemetry-collector-configmap
+            readOnly: true
+          - mountPath: c:/var/lib/swo/checkpoints
```

Comment:
So, are we using forward slashes or backward ones?

Reply:
It doesn't matter, both are functional.  I wanted to preserve one configuration, that's why I did not changed checkpoint option.